### DISCUSSION TITLE
Installation?

### DISCUSSION THREAD

**Discussion by munibsheraz** [AdotBdot/SpicetifyManagerApp#2] [Q&A] [answered]
Hi, I'm new to this so I have no idea what to do with github to install this. I just recently figured out how to change themes (1 hour ago) , but have no idea how to install this GUI application. Sorry for being a noob!

**Comment by AdotBdot** [comment, marked as answer]
Sorry for being late but it's my first public project and I just discovered the Discussions tab. So, all you need to do is to go to the <a href="https://github.com/AdotBdot/SpicetifyManagerApp/releases">Releases</a> page, download latest zip file (currently SpicetifyManager-v1.1.0-x64.zip) and extract to the desired directory.
